### ISSUE TITLE
WebDriverAgent: WebDriverAgentLib: does not build with Appium 1.16-beta.1

### ISSUE THREAD

**Issue by amedvedjev** [appium/appium#13360]
## The problem

with latest appium can't build WebDriverAgent with xCode 11.0

## Environment

* Appium version (or git revision) that exhibits the issue: 1.16-beta.1

## Details

Needs source fixes:

```
controller.autocorrectionEnabled = value;
controller.predictionEnabled = value;
// ->
controller.autocorrectionEnabled = ((NSNumber*)value).boolValue;
controller.predictionEnabled = ((NSNumber*)value).boolValue;
```

```
self.coordinates = CGPointMake(strX.doubleValue, strY.doubleValue);
// ->
self.coordinates = CGPointMake(strX.floatValue, strY.floatValue);
```

**Comment by mykola-mokhnach** [comment]
Is there something that prevents you from creating a PR?

**Comment by mykola-mokhnach** [comment]
strange, I'm on Xcode 11 GM and don't observe any problems

**Comment by KazuCocoa** [comment]
Me, too. Xcode 11.0, .1 and .2 beta also worked... 🤔

**Comment by amedvedjev** [comment]
I am not alone 
https://discuss.appium.io/t/appium-1-16-0-beta1-webdriveragentrunner-build-failed-due-to-incompatible-pointer-to-integer-conversion/28112/2

**Comment by KazuCocoa** [comment]
@amedvedjev 
Could you try to build this branch https://github.com/appium/WebDriverAgent/pull/234 ?
This is only for the pointer error. On my and Appium CI (Xcode 11 env) has no error for it, so I cannot check it.

**Comment by KazuCocoa** [comment]
I've published a new version that addressed `autocorrectionEnabled` and `predictionEnabled`. It is available via `appium@beta`.
@amedvedjev it is appreciated if you can check whether it can build because we could not reproduce it.

It does not have `self.coordinates` change.

**Comment by amedvedjev** [comment]
@KazuCocoa i checked latest webDriverAgent: head 2.0.2
it still have 2 Errors
 - first is second issues i mentioned
 - second if same float problem but another line of code (just forgot to mention)

so left issues are now:
```
// WebDriverAgentLib/Utilities/FBActiveAppDetectionPoint.m:23:70: Implicit conversion loses floating-point precision: 'double' to 'CGFloat' (aka 'float')
CGFloat pointDistance = MIN(screenSize.width, screenSize.height) * 0.2;
// WebDriverAgentLib/Utilities/FBActiveAppDetectionPoint.m:79:52: Implicit conversion loses floating-point precision: 'double' to 'CGFloat' (aka 'float')
self.coordinates = CGPointMake(strX.doubleValue, strY.doubleValue);
```

**Comment by mykola-mokhnach** [comment]
CGFloat might be mapped to float on 32-bit platforms (which we don't want to support). What device you are trying to compile WDA for?

**Comment by amedvedjev** [comment]
General ios device.

**Comment by KazuCocoa** [comment]
The fix is available in `appium@beta` (Published it as appium-webdriveragent 2.0.3)
